okapi-curl

```diff
@@ -10,12 +10,33 @@ if [ $# = 0 ]; then
 	exit 1;
 fi
 
-. ~/.okapi
+# Save off environment variables
```

Comment:
One feels there ought to be a better way to do this ... but I can't think of one.